apps/backend/src/libs/modules/server-application/base-server-application-api.ts

```diff
@@ -29,10 +29,11 @@ class BaseServerApplicationApi implements ServerApplicationApi {
 	}
 
 	public generateDoc(title: string): ReturnType<typeof swaggerJsdoc> {
-		const isProduction =
-			this.config.ENV.APP.ENVIRONMENT === AppEnvironment.PRODUCTION;
+		const isDeployedApplication =
```

Comment:
```suggestion
		const isDevelopmentApplication = this.config.ENV.APP.ENVIRONMENT === AppEnvironment.LOCAL;
```
can we do something like this?